plugins/modules/cloud/docker/docker_image.py

```diff
@@ -737,11 +743,14 @@ def build_image(self):
         if self.target:
             params['target'] = self.target
 
+        build_output = ""
```

Comment:
Could you please change this back to `[]` and do a `'\n'.join()` at the end? That's **a lot** more efficient than string concatenation.

Reply:
Yeah makes sense.